pages/index.tsx

```diff
@@ -2,24 +2,28 @@ import React from "react";
 import {NextPage} from 'next';
 import scss from '../pageStyles/index.module.scss'
 import Layout from "../components/Layout/layout";
-import Link from 'next/link'
-
-const Home: NextPage = () =>  
-    <Layout>
-        <section className={scss.content}>
-            <h1 className={scss.welcome} data-testid='page-render-test'>Welcome to the Test Zone</h1>
-                <h2 className={scss.instructionsHeader}>Test Instructions</h2>
-                <ul className={scss.instructions}>
-                    <li>You must complete the test in one sitting</li>
-                    <li>There is no time limit for this test, but time taken will be reviewed as part of our assessment.</li>
-                    <li>When you are ready and sure you won’t be disturbed, please click “Start”</li>
-                </ul>
-            <Link href="/testlibrary">
-                <a className="buttonYellow" id="buttonYellow">Start Test</a>
-            </Link>
-        </section>
-    </Layout>;
+import QueryToken from "../components/QueryToken/QueryToken";
+
+
+
+const Home: NextPage = () =>
+        <Layout>
+                <section className={scss.content}>
+                    <h1 className={scss.welcome} data-testid='page-render-test'>Welcome to the Test Zone</h1>
+                    <h2 className={scss.instructionsHeader}>Test Instructions</h2>
+                    <ul className={scss.instructions}>
+                        <li>You must complete the test in one sitting</li>
+                        <li>There is no time limit for this test, but time taken will be reviewed as part of our assessment.</li>
+                        <li>When you are ready and sure you won’t be disturbed, please click “Start”</li>
+                    </ul>
+                    <QueryToken href={"/testlibrary"}>
+                        <a className="buttonYellow" id="buttonYellow">Start Test</a>
+                    </QueryToken>
+                </section>
+        </Layout>;
```

Comment:
Looks like we've changed lots of the whitespace here - could we change it back so that its just our wrapper that changes?

Reply:
removed